.gitignore

```diff
@@ -6,7 +6,6 @@
 *.sconsign.dblite
 *.org.eclipse.cdt.core.prefs
 *.language.settings.xml
-*.cproject
```

Comment:
need to double check this change

Reply:
Will remain as ignored. Every developer that would like to push his changes here will uncomment *.cproject, push his changes and again comment *.cproject